docs/modules/ROOT/pages/advanced-how-tos/testing-releasing-single-component.adoc

```diff
@@ -0,0 +1,200 @@
+or by using the HAC UI.
+
+* Update the custom resource by adding the parameter *SINGLE_COMPONENT* with a value of *true*
+
+[source,yaml]
+.integration-test-scenario.yaml
+----
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  name: grouping-enterprise-contract-single-component
+  namespace: shebert-tenant
+spec:
+  application: grouping
+  contexts:
+    - description: Application testing
+      name: application
+  params:
+    - name: SINGLE_COMPONENT
+      value: 'true'
+  resolverRef:
+    params:
+      - name: url
+        value: 'https://github.com/konflux-ci/build-definitions'
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipelines/enterprise-contract.yaml
+    resolver: git
+----
+
+[NOTE]
+====
+You need to apply the change using *kubectl*
+====
+
+* Alternatively, you may edit the Integration Test Scenario using the HAC UI as follows:
+
+image::single-component-integration-test-scenario.png[role="border" alt="Editing your Integration Test Scenario"]
+
+=== Enabling Single Component Mode for Releases
+
+Single Component Mode can be enabled for Releases by making a change to the ReleasePlanAdmission custom resource.
+
+* Update the custom resource by adding the field *singleComponentMode* under _.spec.data_ with a value of *true*
+
+[source,yaml]
+.release-plan-admission.yaml
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  name: single-component-rpa
+  namespace: managed-release-team-tenant
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+spec:
+  applications:
+    - grouping
+  data:
+    mapping:
+      components:
+        - name: comp1
+          repository: quay.io/scoheb/comp1-release
+          tags:
+            - latest
+        - name: comp2
+          repository: quay.io/scoheb/comp2-release
+          tags:
+            - latest
+    singleComponentMode: true
+  origin: shebert-tenant
+  pipeline:
+    pipelineRef:
+      params:
+        - name: url
+          value: 'https://github.com/konflux-ci/release-service-catalog.git'
+        - name: revision
+          value: production
+        - name: pathInRepo
+          value: pipelines/push-to-external-registry/push-to-external-registry.yaml
+      resolver: git
+    serviceAccountName: release-service-account
+    timeouts:
+      pipeline: 4h0m0s
+  policy: e2e-single-component-policy
+----
+
+[NOTE]
+====
+You need to apply the change using *kubectl*
+====
+
+=== Analyzing Test Results and Releases in Single Component Mode
+
+* Once your changes have been applied, you can verify that the mode is in fact activated.
+
+==== Verifying Enterprise Contract Integration Test Scenario results
+
+* Navigate to the logs of your Integration Test Scenario pipeline.
+* Locate the *reduce* step in the log.
+* It should resemble the following output:
+
+[source]
+--
+[reduce] Single Component mode? true
```

Comment:
For this and the next snippet, is the content relevant or just the fact that the reduce parts ran? It might be simpler to parse if we remove the unnecessary details specific to your run.

Reply:
removed some of the snippet